### PR TITLE
blog about new container names

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: _posts/2024-08-02-new-container-names.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+date: 2024-08-02
+github_username: rwaffen
+category: changelog
+---
+
+Beginning of today we are also releaseing our puppetserver and puppetdb containers with shorter names.
+
+The current names are: `voxpupuli/container-puppetserver` and `voxpupuli/container-puppetdb`.
+
+- <https://hub.docker.com/r/voxpupuli/container-puppetserver>
```
Comment (bastelfreak) [inline _posts/2024-08-02-new-container-names.md:13]: the link rendering was a bit tricky in the past, I don't know if this properly renders to hyperrefs. If not we can update it to `[hub.docker.com/r/voxpupuli/container-puppetserver](https://hub.docker.com/r/voxpupuli/container-puppetserver)`

File: _posts/2024-08-02-new-container-names.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- <https://hub.docker.com/r/voxpupuli/container-puppetdb>
+
+The new, shorter names are: `voxpupuli/puppetserver` and `voxpupuli/puppetdb`.
+
+- <https://hub.docker.com/r/voxpupuli/puppetserver>
+- <https://hub.docker.com/r/voxpupuli/puppetdb>
+
+The content is exactly the same, only the name has changed. Only new tags will be pushed to the new names. The old names will also be updated.
+
+We plan to deprecated the old names in 6 months (2025-02), so please update your scripts and configurations to use the new names.
```
Comment (bastelfreak) [inline _posts/2024-08-02-new-container-names.md:23]:
```suggestion
We plan to deprecate the old names in 6 months (2025-02), so please update your scripts and configurations to use the new names.
```

File: _posts/2024-08-02-new-container-names.md (hunk 1 above)
Comment (rwaffen) [reply _posts/2024-08-02-new-container-names.md:15]: this is official markdown, but i can change it to this longer syntax

Comment (rwaffen) [reply _posts/2024-08-02-new-container-names.md:15]: people in the internet say it should work this way
> try surrounding the link with angle brackets like <http://example.com>. Jekyll uses Kramdown, this is from their docs

i want to try this and see whats happen

**Pull request merged**